.github/workflows/proof_ci.yaml

```diff
@@ -0,0 +1,188 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+# CBMC starter kit 2.9
+name: Run CBMC proofs
```

Comment:
Should I remove the old "CBMC Proofs" Codebuild jobs once this is merged?

Reply:
I suggest we keep them running in parallel for a little while to bake and ensure that the new one works smoothly. I'll be working on migrating the data from the old accounts. Once that's done, I can ping you to set the new check as "Required" and remove the old webhook.

Reply:
To clarify, by "a little while" I expect ~3 weeks or so to do the data transfer.